specification/schemas/ReturnMethod.json

```
@@ -0,0 +1,40 @@
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/ReturnMethodResource"
+    },
+    {
+      "properties": {
+        "attributes": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "return_type": {
+              "type": "string",
+              "enum": [
+                "self-shipment",
+                "self-service",
```

This was in the requirements, but in the implementation we chose for the less ambiguous:
```suggestion
                "store-drop-off",
```